signac/contrib/project.py

```diff
@@ -3040,7 +3052,7 @@ def init_project(name, root=None, workspace=None, make_dir=True):
 
     Parameters
     ----------
-    name : str
+    name : str, optional
         The name of the project to initialize.
     root : str
```

Comment:
For consistency, mark all optional parameters here and below:
```suggestion
    root : str, optional
```